test/src/loaders/gtdb/gtdb_genome_attribs_loader_test.py

```diff
@@ -56,16 +56,15 @@ def test_create_json_default(setup_and_teardown):
     command = ['python', script_file,
                os.path.join(caller_file_dir, 'SAMPLE_ar53_metadata_r207.tsv'),
                os.path.join(caller_file_dir, 'SAMPLE_bac120_metadata_r207.tsv'),
-               '--load_version', load_version,
+               '--load_ver', load_version,
                '--kbase_collection', kbase_collections,
                '-o', result_file]
 
     _exe_command(command)
 
     expected_docs_length = 20
-    expected_doc_keys = {'_key', 'coll', 'load_ver', 'genome_name', 'checkm_completeness',
-                         'high_checkm_marker_count', 'trna_selenocysteine_count',
-                         'n50_scaffolds'}  # cherry-pick a few from gtdb_genome_attribs_helper.EXIST_FEATURES
+    expected_doc_keys = {'_key', 'coll', 'load_ver', 'accession', 'checkm_completeness',
```

Comment:
Ok, so we're changing `genome_name` -> `accession` in the arango docs?

Reply:
Do you think that's a good idea? I remember we were talking about leaving what GTDB provided as is.

Reply:
This particular field is used in the `genome_attributes` data product so it'd need to be updated there.

Reply:
https://github.com/kbase/collections/blob/main/src/common/storage/collection_and_field_names.py#L71

Reply:
👍 you are right. Renamed `FLD_GENOME_ATTRIBS_GENOME_NAME` to `FLD_GENOME_ATTRIBS_ACCESSION`.

Reply:
So there's another hitch here - this is used as the default sort order for the genome attributes when a sort order isn't specified, which is going to happen every time a user hits the UI for the first time or when they change other settings but don't specify the sort order. That's why the data product service code sets up an index for this field - so that the initial load will be fast. 

That means whatever field we pick as the default should exist (and ideally be unique) for all current and future collections. Is that true for the accession field?

Reply:
For GTDB (I checked r207 and r202), `accession` is unique. But once we onboard other data sets, say PMI, I don't think that field will exist.

Reply:
I think we want to have the default sort field a field that is 1) guaranteed to exist, and 2) ideally unique (but it doesn't 100% have to be).

I'm open to ideas, but one possibility is that we go back to the genome_name field and have all attributes loaders use that field for some sort of genome name - for GTDB it could be the accession, for PMI it could be whatever ID they use or worst case the workspace object name / ID, etc

Reply:
👍 makes sense to me. Revert to using FLD_GENOME_ATTRIBS_GENOME_NAME. And added a check to ensure that field does exist before converting to the Json doc file.

Reply:
Sorry to keep going back and forth on this. Looking at the current changes:

* I think we should keep the accession field, but just copy that field to the new field name. 
* I'm not sure genome_name is the best field name. The contents might not be a name, and might collide with other collections' attributes. Maybe one of
  * `genome_id`
  * `internal_genome_id`
  * `kbase_id`
  * `kbase_genome_id`
  * `kbase_assigned_id`
  * `kbase_assigned_genome_id`

... or something like that.

Reply:
👍  using `kbase_genome_id` as the sort key.

Reply:
:+1: